index1.html

```diff
@@ -0,0 +1,104 @@
+ <title>Zoum-Capstone-project</title>
+</head>
+<body class="body">
+<div class="firstFace">  </div>    
+
+ <header>
+       <div class="toggle">
+              <i class="fas fa-bars ouvrir"></i> 
+              <i class="fas fa-times fermer"></i>         
+          </div>
+          <div class="menu">
+              <li class="content-liA"><a href="index1.html">About</a></li>
+              <li class="content-liA"><a href="#">program</a></li>
+              <li class="content-liA"><a href="#">Join</a></li>
+              <li class="content-liA"><a href="#">Sponsor</a></li>
+              <li class="content-liP"><a href="#">News</a></li>
+              <li class="content-liC"><a href="#">CC compaign</a></li>
+         
+          </div>
+
+          <div class="header-desk">
+              <div class="header-first">
+                     <li><a href="#" target="_blank"><i class="fab fa-facebook"></i></a></li>
+                     <li> <a href="https://twitter.com/zoumaniguimass1" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a></li>
+                     <ul class=" nav-headerfirst">
+                            <li><a href="#">English</a></li>
+                            <li><a href="#">My page</a></li>
+                            <li><a href="#">loyout</a></li>
+                     </ul>
+              </div>
+              <div class="header-second">
+               <div class="logo"><a href="index.html"><img src="images/icon.png " width="100%" height="80%" alt=""></a></div>
+               <nav>
+                      <ul>
+                             <li class="content-liA"><a href="index1.html">About</a></li>
+                             <li class="content-liA"><a href="#">program</a></li>
+                             <li class="content-liA"><a href="#">Join</a></li>
+                             <li class="content-liA"><a href="#">Sponsor</a></li>
+                             <li class="content-liP"><a href="#">News</a></li>
+                             <li class="content-liCc"><a href="#">CC compaign</a></li>
+                      </ul>
+               </nav>
+        </div>
+              </div>
+          </div>
+ </header>
+
+ <div class="welcom">
+  <h1 class="welcom-h1">Developed by young developers for the improvement of national education</h1>
+
+  <p class="welcom-p1">
+      
+  <p class="welcom-p2">
+   The integration of Artificial Intelligence (AI) in education assessment is a cutting-edge technological advancement that is revolutionizing the traditional way of correcting exams. With the use of AI, the process of grading exams becomes faster, more accurate, and less subjective. This technology uses algorithms that are programmed to recognize patterns and identify correct answers based on a set of criteria.
+  </p>
+  <p class="welcom-p3">
+     <span>Please contact us for more information and investment  <a href="#">zoumaniguimass12@gmail.com</a></span>
+   
+  </p>
```

Comment:
- [ ] Kindly follow the given design guideline for this section for both mobile and desktop versions.
  - Every content should be `centered` to the body for both mobile and desktop versions.
  - Kindly don't use bold text for the `Please contact us` section for the mobile version.
 
- Given Design:
![image](https://user-images.githubusercontent.com/60083437/227038241-97a1fc84-4000-471d-9bbe-e71d088971f3.png)